packages/workers-shared/asset-server-worker/src/global.d.ts

```diff
@@ -0,0 +1,32 @@
+type Environment = "production" | "local";
+
+type Env = {
+	ASSETS_MANIFEST: ArrayBuffer;
+	ASSETS_KV_NAMESPACE: KVNamespace;
+	ENVIRONMENT: Environment;
+};
+
+type BodyEncoding = "manual" | "automatic";
+
+interface ResponseInit {
+	encodeBody?: BodyEncoding;
+}
+
+interface KVNamespace {
```

Comment:
Why can't we use `@cloudflare/workers-types` here? That would save manually typing these things and potentially making mistakes/getting out of sync.

Reply:
you are totes right! removed!